omod/src/main/webapp/resources/htmlforms/sierra_leone/vitals.xml

```diff
@@ -0,0 +1,784 @@
+<!--
+  ~ The contents of this file are subject to the OpenMRS Public License
+  ~ Version 1.0 (the "License"); you may not use this file except in
+  ~ compliance with the License. You may obtain a copy of the License at
+  ~ http://license.openmrs.org
+  ~
+  ~ Software distributed under the License is distributed on an "AS IS"
+  ~ basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing rights and limitations
+  ~ under the License.
+  ~
+  ~ Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+  -->
+
+<htmlform id="vitals-form" formUuid="68728aa6-4985-11e2-8815-657001b58a90"
+          formName="Vitals" formEncounterType="4fb47712-34a6-40d2-8ed3-e153abbd25b7"
```

Comment:
Are these UUIDs the same as the other vitals forms? I think that they should be -- at least the `formEncounterType`.

Reply:
Just copy pasted from original vitals.xml file in html forms folder - so if they need changing let me know